src/main/java/org/folio/helper/ExpectHelper.java

```diff
@@ -0,0 +1,147 @@
+package org.folio.helper;
+
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+import one.util.streamex.StreamEx;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.rest.core.models.RequestContext;
+import org.folio.rest.jaxrs.model.ExpectCollection;
+import org.folio.rest.jaxrs.model.ExpectPiece;
+import org.folio.rest.jaxrs.model.Piece;
+import org.folio.rest.jaxrs.model.ProcessingStatus;
+import org.folio.rest.jaxrs.model.ReceivingResult;
+import org.folio.rest.jaxrs.model.ReceivingResults;
+import org.folio.rest.jaxrs.model.ToBeExpected;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.mapping;
+import static java.util.stream.Collectors.toList;
+
+public class ExpectHelper extends CheckinReceivePiecesHelper<ExpectPiece> {
```

Comment:
Can we rename to PieceService without extend operation with method moveToExpected(ExpectCollection expectCollection)

Reply:
there is method for acq unit check in base class, also for future we potentially can be needed additional checks like to u[date statuses etc, so we decided to remain extends from CheckinReceivePiecesHelper